<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Auto_Widget</class>
 <widget class="QWidget" name="Auto_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_48" stretch="10,1">
     <item>
      <widget class="QCustomPlot" name="customPlot_3" native="true"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QCheckBox" name="checkBox_current_3">
           <property name="text">
            <string>current</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_reference_3">
           <property name="text">
            <string>reference</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>208</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_49" stretch="4,1,2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_50">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_51">
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>方位:</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_cur_azm_4">
             <property name="text">
              <string>000.00</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_52">
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>极化:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_cur_pol_4">
             <property name="text">
              <string>000.00</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_53">
           <item>
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>俯仰:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_cur_el_4">
             <property name="text">
              <string>000.00</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_54">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_55">
           <item>
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>预置极化:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_per_pol_4">
             <property name="text">
              <string>000.00</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_56">
           <item>
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>预置俯仰:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_per_el_4">
             <property name="text">
              <string>000.00</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_57">
           <item>
            <widget class="QLabel" name="label_24">
             <property name="text">
              <string>预置方位:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_per_azm_4">
             <property name="text">
              <string>000.00</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_58" stretch="2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_59">
           <item>
            <widget class="QPushButton" name="pushButton_19">
             <property name="text">
              <string>停止</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_20">
             <property name="text">
              <string>预留</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_60" stretch="2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_61">
           <item>
            <widget class="QPushButton" name="pushButton_21">
             <property name="text">
              <string>收藏</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_22">
             <property name="text">
              <string>展开</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_62" stretch="0">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_63">
           <item>
            <widget class="QPushButton" name="pushButton_23">
             <property name="text">
              <string>复位</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_24">
             <property name="text">
              <string>对星</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
